src/index.tsx

```diff
@@ -28,11 +28,11 @@ const App = () => {
           </Switch>
         </PageWrapper>
       </Router>
-    </React.StrictMode>
+    </React.Fragment>
```

Comment:
I removed StrictMode because some components used by amplify does not handle StrictMode

Reply:
It seems that I don't have the problem anymore because we have switched back to aws-aplify-react rather than @aws-aplify/ui-react, so I'll switch back to StrictMode.
Basically the problem seems to come from StyledComponents used by @aws-aplify/ui-react -> https://lifesaver.codes/answer/react-strictmode-throws-warning-because-of-finddomnode-use